### PR TITLE
Feature/audio manager

### PR DESCRIPTION
**Pull request by ElecJason**

### CODE REVIEW

File: assets/Scripts/Settings/AudioHandeler.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+public class AudioHandler : MonoBehaviour
```
Comment (thomasberrens) [inline assets/Scripts/Settings/AudioHandeler.cs:4]: please change the file name to "AudioHandler", currently it is "AudioHandeler"

File: assets/Scripts/Settings/AudioHandeler.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+public static class AudioManager
+{
+    public static float volume = 1.0f;
+
+    static AudioManager()
+    {
+        // Set the initial volume for all AudioSources in the scene
+        /*foreach (var audioSource in FindObjectsOfType<AudioManager>())
+        {
+            audioSource.volume = volume;
```
Comment (thomasberrens) [inline assets/Scripts/Settings/AudioHandeler.cs:29]: remove this block of code if it isn't necessary

File: assets/Scripts/Sound/SoundManager.cs [hunk 3]
```diff
@@ -0,0 +1,30 @@
+using UnityEngine;
+
+public class SoundManager : MonoBehaviour
+{
+    private static SoundManager instance;
```
Comment (thomasberrens) [inline assets/Scripts/Sound/SoundManager.cs:5]: please avoid using instances for MonoBehaviours, the game manager is most of the times the only one who has a static instance. You can make this a serializablefield variable in the GameManager class, and if you want to acces the soundmanager you can do it by GameManager.Instance.SoundManager

**Review by deanhendriks — APPROVED** [review]

**Review by thomasberrens — CHANGES_REQUESTED** [review]
some small changes need to be made (check pull request feedback) but if that is done it is fine

File: assets/Scripts/Settings/AudioHandeler.cs (hunk 1 above)
Comment (ElecJason) [reply assets/Scripts/Settings/AudioHandeler.cs:4]: removed this entirely, was an old script

File: assets/Scripts/Settings/AudioHandeler.cs (hunk 2 above)
Comment (ElecJason) [reply assets/Scripts/Settings/AudioHandeler.cs:29]: removed script entirely

**Pull request merged**